src/main/java/org/jitsi/protocol/xmpp/util/SimulcastGrouping.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
+ *
+ * Copyright @ Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.protocol.xmpp.util;
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.colibri.*;
+import net.java.sip.communicator.impl.protocol.jabber.extensions.jingle.*;
+
+import java.util.*;
+
+/**
+ * Utility class for the simulcast {@link SourcePacketExtension} grouping.
+ */
+public class SimulcastGrouping
```

Comment:
why not have this extend `SourceGroupPacketExtension` ? Or `SourceGroup`?

Reply:
SourceGroupPacketExtension is a packet extension where this one is a logical wrapper around simulcast grouping which consists of SIM + multiple FID groups

Reply:
Packet extension is a Smack's thing and we used to only store there fields and attributes of the XML elements.

Reply:
what about `SourceGroup`?